feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -315,6 +318,80 @@ class BillingWrapperTest {
         )
     }
 
+    @Test
+    fun `obfuscatedAccountId is set for non-transfer purchases`() {
+        setup()
+        val appUserID = "jerry"
+
+        val skuDetails = stubSkuDetails(productId = "product_a")
+        val activity: Activity = mockk()
+
+        mockkStatic(BillingFlowParams::class)
+        val mockBuilder = mockk<BillingFlowParams.Builder>(relaxed = true)
+        every {
+            BillingFlowParams.newBuilder()
+        } returns mockBuilder
+
+        every {
+            mockBuilder.setSkuDetails(any())
+        } returns mockBuilder
+
+        every {
+            mockClient.launchBillingFlow(any(), any())
```

Comment:
we had also mocked the params:

```
val params = mockk<BillingFlowParams>(relaxed = true)
        every {
            mockBuilder.build()
        } returns params

 every {
            mockClient.launchBillingFlow(any(), params)
```

but that didn't seem necessary anymore?

Reply:
`mockClient.launchBillingFlow(any(), params)` is better than `mockClient.launchBillingFlow(any(), any())` because it's more specific and it's actually verifying that `mockClient.launchBillingFlow` is called with the right params. When using `params` instead of `any()`, `mockClient.launchBillingFlow` would fail with a "not mocked error" if different params were ever sent to the method.